src/Core/Config/Capability/IManageConfigValues.php

```diff
@@ -78,6 +89,15 @@ public function get(string $cat, string $key, $default_value = null, bool $refre
 	 */
 	public function set(string $cat, string $key, $value): bool;
 
+	/**
+	 * Creates a transactional config value store, which is used to set a bunch of values at once
+	 *
+	 * It relies on the current instance, so after save(), the values of this config class will get altered at once too.
+	 *
+	 * @return ISetConfigValuesTransactional
+	 */
+	public function transactional(): ISetConfigValuesTransactional;
```

Comment:
I'd rather we use the existing PHP vocabulary for transactions, namely 
```suggestion
	public function beginTransaction(): ISetConfigValuesTransactional;
```